### PR TITLE
Feature home page

### PR DESCRIPTION
**Pull request by MHP200**
Files change:
- BecomeTutor.jsx
- HomePage.jsx
- NavBar.jsx
- App.jsx
assest/image (added 2 image):
- Background_Image.png
- Logo_Image.png

Note:
On the homepage, if the user clicks on the "Tutors" link, it doesn't take them to the tutor profile because it hasn't been implemented yet.

### CODE REVIEW

File: my-react-app/src/components/HomePage.jsx [hunk 1]
```diff
@@ -7,91 +7,106 @@ import image1 from '../assets/images/Polygon1.svg';
...
   const results = useSelector(state => state.results);
   const dispatch = useDispatch();
-  
-  const handleSearch = (query) => {
+
+  const handleSearch = query => {
     dispatch(searchAsync(query));
   };
 
+  const tutorProfiles = [
```
Comment (John42506176Linux) [inline my-react-app/src/components/HomePage.jsx:63]: I'd move this to the constants folder just create a file called tutorProfiles.tsx and move this there, then just import it. I'd rather keep this file clean.

File: my-react-app/src/components/BecomeTutor.jsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+                  onChange={(e) => setResume(e.target.files[0])}
+                />
+              </div>
+              {errors.resume && <span style={{ color: "red" }}>This field is required.</span>}
+            </div>
+            <div style={formGroupStyle}>
+              <label>*Topic:</label>
+              <div>
+                <input
+                  type="text"
```
Comment (John42506176Linux) [inline my-react-app/src/components/BecomeTutor.jsx:84]: For all of your inputs you should give them a class, with your button styles, I'd suggest creating a directory called styles, with a file called styles.jsx and put all of your styles there. This was you can keep it all organized, and use the same button styling for all files. Make sure to show me the design on figma first, for the buttons, before you start writing the code first.

File: my-react-app/src/components/BecomeTutor.jsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  const checkmarkStyle = {
+    color: "green",
+    fontWeight: "bold",
+  };
+
+  return (
+    <div>
+      <NavBar /> {/* Render the Navbar component at the top of the page */}
+      <div className="content">
```
Comment (John42506176Linux) [inline my-react-app/src/components/BecomeTutor.jsx:65]: There should be some space here write now, it's pressed up against the navbar

File: my-react-app/src/components/HomePage.jsx (hunk 1 above)
Comment (MHP200) [reply my-react-app/src/components/HomePage.jsx]: pls check the new code

File: my-react-app/src/components/BecomeTutor.jsx (hunk 2 above)
Comment (MHP200) [reply my-react-app/src/components/BecomeTutor.jsx]: pls check the new code

File: my-react-app/src/components/BecomeTutor.jsx (hunk 3 above)
Comment (MHP200) [reply my-react-app/src/components/BecomeTutor.jsx]: pls check the new code

**Review by John42506176Linux — APPROVED** [review]

**Pull request merged**